scripts/scil_recognize_single_bundle.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+    Compute a simple Recobundles (single-atlas & single-parameters).
+    The model need to be cleaned and lightweight.
+    Transform should come from ANTs: (using the --inverse flag)
+    AntsRegistration -m MODEL_REF -f SUBJ_REF
+    ConvertTransformFile 3 0GenericAffine.mat 0GenericAffine.npy --ras --hm
+"""
+
+import argparse
+import pickle
+
+from dipy.segment.bundles import RecoBundles
+from dipy.tracking.streamline import transform_streamlines
+import nibabel as nib
+from nibabel.streamlines.array_sequence import ArraySequence
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exists)
+
+
+def buildArgsParser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+
+    p.add_argument('in_tractogram',
+                   help='Input tractogram filename (trk or tck).')
+    p.add_argument('in_model',
+                   help='Model to use for recognition (trk or tck).')
+    p.add_argument('transformation',
+                   help='4x4 transformation to bring the model onto \n'
+                   'the input tractogram')
+    p.add_argument('output_name',
+                   help='Output tractogram filename.')
+
+    p.add_argument('--wb_clustering_thr', type=float, default=8,
+                   help='Clustering threshold used for the whole brain '
+                   '[%(default)smm].')
+    p.add_argument('--model_clustering_thr', type=float, default=4,
+                   help='Clustering threshold used for the model '
+                   '[%(default)smm].')
+    p.add_argument('--pruning_thr', type=float, default=6,
+                   help='MDF threshold used for final streamlines selection '
+                   '[%(default)smm].')
+
+    p.add_argument('--slr_threads', type=int, default=None,
+                   help='Number of threads for SLR [all].')
+    p.add_argument('--seed', type=int, default=None,
+                   help='Random number generator seed [%(default)s].')
+    p.add_argument('--inverse', action='store_true',
+                   help='Use the inverse transformation.')
```

Comment:
Maybe add this option must be used if the transform come from ANTs

Reply:
Well, this can be as confusing. It also depend on how you called ANTs.
Depending on the moving/target option used with ANTs, maybe you don't need it. (e.g A->B could require --inverse, but not B->A)

Reply:
I agree. forget this comment